### ISSUE TITLE
Please release for Scala 2.13.3

### ISSUE THREAD

**Issue by japgolly** [scalameta/scalameta#2081]
Thank you!

**Comment by olafurpg** [comment]
The latest release already supports 2.13.3. Can you elaborate?

**Comment by japgolly** [comment]
Are you sure? I'm using sbt-scalafix 0.9.17 and I have this in my SBT:

```scala
      ThisBuild / semanticdbEnabled := true,
      ThisBuild / semanticdbVersion := "4.3.15",
```

but I get this error:

```
[warn] 	Note: Unresolved dependencies path:
[warn] 
[error] (base-util-jvm / update) sbt.librarymanagement.ResolveException: Error downloading org.scalameta:semanticdb-scalac_2.13.3:4.3.15
[error]   Not found
[error]   Not found
[error]   not found: /home/golly/.ivy2/local/org.scalameta/semanticdb-scalac_2.13.3/4.3.15/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scalameta/semanticdb-scalac_2.13.3/4.3.15/semanticdb-scalac_2.13.3-4.3.15.pom
```

The Scala 2.13.3 PR was merged after 4.3.15 and there hasn't been a release since.

**Comment by olafurpg** [comment]
4.3.16 has been released with 2.12.3 support. I haven’t published a GitHub release yet

**Comment by japgolly** [comment]
Ah ok, awesome. Thanks as always Olar!
